templates/zerver/config_error.html

```diff
@@ -0,0 +1,41 @@
+{% extends "zerver/portico.html" %}
+
+{% block customhead %}
+{{ super() }}
+<meta http-equiv="refresh" content="60;URL='/'">
+{% endblock %}
+
+{% block portico_content %}
+
+<div class="error_page" style="padding-bottom: 60px;">
+    <div class="container">
+        <div class="row-fluid">
+            <img src="/static/images/500art.svg" alt=""/>
+            <div class="errorbox">
+                <div class="errorcontent">
+                    <h1 class="lead">Configuration error</h1>
+                    <p>Seems like you haven't properly configured authentication!</p>
+                    <p>Try restarting the development environment, as it is possible some changes you made require a restart.</p>
+                    <br/>
+                    {% if google_auth_enabled %}
```

Comment:
I think you want these conditionals to be checking a context element like "google_auth_misconfigured" that's passed into the template directly in the `HttpResponseRedirect` call and actually knows what mistake the user made...